Automattic/jetpack
# [RNMobile] Fix inserting item from media library

Closes https://github.com/wordpress-mobile/gutenberg-mobile/issues/5572.

**⚠️ This PR depends on the following PRs:**
- https://github.com/WordPress/gutenberg/pull/48994
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/5558
- https://github.com/wordpress-mobile/WordPress-Android/pull/18106
- https://github.com/wordpress-mobile/WordPress-iOS/pull/20326

## Proposed changes:

* When inserting an item from the media library, the VideoPress ID is passed via the `metadata` parameter of Media. Before this PR we were getting the value directly from the Media object.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






**NOTE:** For testing this change it's required to use the changes from the PRs referenced above.

1. Open the app.
2. Go to a Simple site _(videos uploaded in Simple sites are automatically added to VideoPress)_
3. Go to the Media screen.
4. Upload a video.
5. Create/open a post.
6. Add VideoPress block.
7. Select "WordPress Media Library" option.
8. Select the uploaded video.
9. Observe that the placeholder is no longer displayed.
10. Preview the post.
11. Observe that the block is displayed and that the video can be played.
